streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/StreamSourceNode.java

```diff
@@ -73,14 +73,16 @@ public String toString() {
     public void writeToTopology(final InternalTopologyBuilder topologyBuilder) {
 
         if (topicPattern().isPresent()) {
-            topologyBuilder.addSource(consumedInternal().offsetResetPolicy(),
+            // TODO mjsax
+            topologyBuilder.addSource(consumedInternal().legacyOffsetResetPolicy(),
```

Comment:
Will revert in follow up PR when `legacyOffsetResetPolicy()` removed, and we move to the new `AutoOffsetReset` class as parameter fro `addSource()` (same below)